src/components/navigation/QuizConceptionView.js

```diff
@@ -0,0 +1,38 @@
+import Box from "@mui/material/Box";
+import {Tab, Tabs} from "@mui/material";
+import {useState} from "react";
+import CreateView from "../create/CreateView";
+import ResultTables from "../results/ResultTables";
+
+const QuizConceptionView = () => {
+
+  const [tab, setTab] = useState(0)
+
+  const TabPanel = ({children, tab, index}) => {
+    return (
+        tab === index && (
+            <Box sx={{pt: 4}}>
+              {children}
+            </Box>)
+    )
+  }
+
+  return (
+      <Box sx={{width: '100%'}}>
+        <Box sx={{borderBottom: 1, borderColor: 'divider'}}>
+          <Tabs value={tab} onChange={(_, v) => setTab(v)}>
+            <Tab label="Create Quiz"/>
+            <Tab label="Results"/>
```

Comment:
To b consistent with the rest of the code, I would ask you to add `t` to translate the text! It uses `useTranslation` I think you can find some examples in the repo^^

Reply:
Ok sure I'll add that.

Should I directly translate it to french as well ?